### PR TITLE
feature - favorite venue

### PR DESCRIPTION
**Pull request by at-phutran**
- UI
- Save the favorite venue
- Load favorite líst from DB
- Delete favorite venue

### CODE REVIEW

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: ATFoody/View/Controller/Home/LocationViewModel.swift [hunk 1]
```diff
@@ -71,6 +91,7 @@ final class LocationViewModel: ViewModel {
         }
     }
 
+
```
Comment (at-daiho) [inline ATFoody/View/Controller/Home/LocationViewModel.swift]: WARNING - Limit vertical whitespace to a single empty line. Currently 2.

File: ATFoody/View/Controller/Home/LocationViewModel.swift [hunk 2]
```diff
@@ -71,9 +89,9 @@ final class LocationViewModel: ViewModel {
```
Comment (at-daiho) [inline ATFoody/View/Controller/Home/LocationViewModel.swift]: WARNING - Limit vertical whitespace to a single empty line. Currently 2.

**Comment by at-daiho** [comment]
Total linter violations in pull request: 2
Serious: 0
Warnings: 2

**Comment by at-daiho** [comment]
Total linter violations in pull request: 3
Serious: 0
Warnings: 3

**Comment by at-daiho** [comment]
:+1: There are no linter violations.

File: ATFoody/View/Controller/Favorite/FavoriteViewModel.swift [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+        realm.write {
+            if let venue = realm.objects(Venue.self).filter("id = '\(id)'").first {
+                venue.isFavorite = false
+            }
+        }
+    }
+}
+
+extension FavoriteViewModel {
+    func numberOfVenueFavorite() -> Int {
```
Comment (at-quangphu) [inline ATFoody/View/Controller/Favorite/FavoriteViewModel.swift]: `numberOfItems()`

File: ATFoody/View/Controller/Favorite/FavoriteViewModel.swift [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+        return venuesFavorite.count
+    }
+
+    func viewModelForRow(at indexPath: IndexPath) -> SearchCellViewModel {
```
Comment (at-quangphu) [inline ATFoody/View/Controller/Favorite/FavoriteViewModel.swift]: `viewModelForItem`

**Pull request merged**